src/tests/component.ts

```diff
@@ -163,6 +163,8 @@ export function describeComponent(description: string,
         baseModules.forEach((baseModule) => beforeEach(angular.mock.module(baseModule)));
 
         beforeEach(angular.mock.module(($provide: angular.auto.IProvideService) => {
+            $provide.value('$rootElement', angular.element(document.body));
```

Comment:
Added.

This was needed because some tests didn't pass unless you added `.parent('body')` to e.g. dialog, which was wrong because you had to alter implementation for tests to pass.